search_service/api/document.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+from http import HTTPStatus
+from typing import Tuple, Any
+
+from flask_restful import Resource, reqparse
+from search_service.proxy import get_proxy_client
+from search_service.models.table import TableSchema
+from search_service.api.table import TABLE_INDEX
+
+LOGGER = logging.getLogger(__name__)
+
+
+class DocumentAPI(Resource):
```

Comment:
could we call TableDocumentAPI as it only update table document? Once we have user document or dashboard document we could either extend it or refactor to be more generic.

Reply:
I can change the name here to `TableDocumentAPI`. I did make the methods in elasticsearch_proxy a bit more generic already, so it should be straightforward to convert this file so that it uses the UserSchema instead of TableSchema when we are ready to take actions on user documents